claim-db-worker/components/Footer.tsx

```
@@ -0,0 +1,68 @@
+import Image from "next/image";
+
+export function Footer() {
+  return (
+    <footer className="w-full box-border overflow-hidden mt-32 md:mt-auto mb-8">
+      <div className="max-w-7xl w-full mx-auto flex justify-between items-center flex-col text-center gap-6 sm:flex-row sm:gap-0 px-4">
+        <a target="_blank" rel="opener noferrer" href="https://prisma.io">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the "noopener" to the existing attribute.


(lint/security/noBlankTarget)